src/main/java/io/temporal/internal/replay/WorkflowExecutor.java

```diff
@@ -24,45 +24,53 @@
 import io.temporal.api.query.v1.WorkflowQuery;
 import io.temporal.api.query.v1.WorkflowQueryResult;
 import io.temporal.api.workflowservice.v1.PollWorkflowTaskQueueResponseOrBuilder;
+import java.time.Duration;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
 public interface WorkflowExecutor {
 
-  WorkflowTaskResult handleWorkflowTask(PollWorkflowTaskQueueResponseOrBuilder workflowTask)
-      throws Throwable;
+  /**
+   * Handles a single workflow task.
+   *
+   * @param workflowTask task to handle
+   * @return true if new task should be created synchronously as local activities are still running.
```

Comment:
[Nit] what does it mean by return "true if new task should be created synchronously"? Does this mean not null?

Reply:
See my earlier comment about force creating workflow tasks.